src/platforms/common/performance/index.mdx

```diff
@@ -94,11 +94,11 @@ Test out tracing by starting and finishing a transaction, which you _must_ do so
 
 While you're testing, set <PlatformIdentifier name="traces-sample-rate" /> to `1.0`, as that ensures that every transaction will be sent to Sentry.
 
-Once testing is complete, **we recommend lowering this value in production** by either lowering your <PlatformIdentifier name="traces-sample-rate" /> value, or switching to using <PlatformIdentifier name="traces-sampler" /> to dynamically sample and filter your transactions.
+Once testing is complete, you may want to consider setting a lower <PlatformIdentifier name="traces-sample-rate" /> value, or switching to using <PlatformIdentifier name="traces-sampler" /> to selectively sample and filter your transactions, based on contextual data. 
```

Comment:
```suggestion
Once testing is complete, you may want to set a lower <PlatformIdentifier name="traces-sample-rate" /> value, or switch to using <PlatformIdentifier name="traces-sampler" /> to selectively sample and filter your transactions, based on contextual data. 
```

In case you want a record of Ian's comments: 
"Lets just say something more balanced >
Like for small applicaitons under X TxNs you may not need any sampling.  For FE apps there is no scale concern and sampling is optional.  For extermely hhigh scale BE apps, you may wish to explore sampling if load on the network or system resources are high."

Reply:
I recommend consolidating this paragraph and the one below it into one.